Security/Security-with-AAD-pod-managed-identities/README.md

```diff
@@ -0,0 +1,279 @@
+
+Now that the storage account has been created and linked to our cluster, we can go ahead and test this feature.
+
+### Accessing the pod from the cluster
+
+1. To try and connect to the storage account, we will deploy a new pod created using the Azure CLI container image. Create a new file in your working directory. You can use your preferred editor.
+
+   ```bash
+   code deployment‑with‑identity.yaml
+   ```
+
+1. Copy the manifest file code below into the file you just created and save it
+
+   ```yaml
+   apiVersion: apps/v1
+   kind: Deployment
+   metadata:
+     name: access-blob
+   spec:
+     selector:
+       matchLabels:
+         app: access-blob
+     template:
+       metadata:
+         labels:
+           app: access-blob
+           aadpodidbinding: access-blob-id
+       spec:
+         containers:
+         - name: azure-cli
+           image: mcr.microsoft.com/azure-cli
+           command: [ "/bin/bash", "-c", "sleep inf" ]
+   ```
+
+1. Create this deployment using the following command
+
+   ```bash
+   kubectl create -f deployment‑with‑identity.yaml
+   ```
+
+1. When the pod is in the *Running* state copy run the code below replacing the items in the angle brackets with the name of the access-blob pod
+
+   ```bash
+   kubectl exec -it <access-blob pod name> -- sh
+   ```
+
+1. Within the container's terminal, enter the command below
+
+   ```bash
+   az login --identity -u <client ID of managed identity> \
+   	--allow-no-subscription -o table
+   ```
+
+1. This should return an output similar to this picture below
+
+   <img src="pictures/managed-identity-enabled.PNG" alt="managed identities" style="zoom:40%;" />
+
+1. Now you can try accessing the blob storage account and downloading the file
+
+   ```bash
+   az storage blob download --account-name <storage account name> \
+   	--container-name <storage container name> --auth-mode login \
+   	--file <filename> --name <filename> -o table
+   ```
+
+   <img src="pictures/upload-success.PNG" alt="managed identities" style="zoom:40%;" />
+
+1. You can exit the container using the *exit* command. Next we will verify that pods that don't have a managed identity linked to the private storage account you created. Create a new manifest file.
+
+   ```bash
+   code deployment-without-identity.yaml
+   ```
+
+1. Copy and paste the code below into the file and save it
+
+   ```yaml
+   apiVersion: apps/v1
+   kind: Deployment
+   metadata:
+     name: no-access-blob
+   spec:
+     selector:
+       matchLabels:
+         app: no-access-blob
+     template:
+       metadata:
+         labels:
+           app: no-access-blob
+       spec:
+         containers:
+         - name: azure-cli
+           image: mcr.microsoft.com/azure-cli
+           command: [ "/bin/bash", "-c", "sleep inf" ]
+   ```
+
+1. Create the new pod using the same image but without access to the managed identity called no-access-blob
+
+    ```bash
+    kubectl create -f deployment-without-identity.yaml
```

Comment:
Are you sure you meant to do `kubectl create` and not `kubectl apply`?

Reply:
@rickrain changed to apply